src/main/java/com/upserve/uppend/metrics/AppendOnlyStoreWithMetrics.java

```diff
@@ -3,12 +3,16 @@
 import com.codahale.metrics.*;
 import com.google.common.collect.Maps;
 import com.upserve.uppend.*;
+import org.slf4j.Logger;
 
+import java.lang.invoke.MethodHandles;
 import java.util.Map;
 import java.util.function.BiConsumer;
 import java.util.stream.Stream;
 
 public class AppendOnlyStoreWithMetrics implements AppendOnlyStore {
+    private static final Logger log = org.slf4j.LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```

Comment:
Why statically importing?

Reply:
That's the pattern for importing the logger in the rest of uppend. It seems reasonable to me.